regression/scripts/Vehicles/Boeing_737.py

```diff
@@ -1,52 +1,53 @@
-# Vehicles.py
-# 
-# Created:  Nov 2015, Carlos / Tarik
-# Modified: Mar 2016, M. Vegh
+# Boeing_737.py
+#
+# Created:  Feb 2017, M. Vegh
+# Modified: 
 
-# ----------------------------------------------------------------------        
+""" setup file for the Boeing 737 vehicle
+"""
+
+
+# ----------------------------------------------------------------------
 #   Imports
-# ----------------------------------------------------------------------    
+# ----------------------------------------------------------------------
 
-import SUAVE
 import numpy as np
-from SUAVE.Core import Units,Data
+import SUAVE
+from SUAVE.Core import Units
 from SUAVE.Methods.Propulsion.turbofan_sizing import turbofan_sizing
 
 
+
+
 # ----------------------------------------------------------------------
 #   Define the Vehicle
 # ----------------------------------------------------------------------
 
-def setup():
-    
-    base_vehicle = base_setup()
-    configs = configs_setup(base_vehicle)
-    
-    return configs 
-    
-    
-def base_setup():
+def vehicle_setup():
     
     # ------------------------------------------------------------------
     #   Initialize the Vehicle
     # ------------------------------------------------------------------    
-
+    
     vehicle = SUAVE.Vehicle()
     vehicle.tag = 'Boeing_737800'    
-
-
+    
+    
     # ------------------------------------------------------------------
     #   Vehicle-level Properties
     # ------------------------------------------------------------------    
 
     # mass properties
-    vehicle.mass_properties.max_takeoff               = 79015.   # kg
-    vehicle.mass_properties.takeoff                   = 79015.   # kg
-    vehicle.mass_properties.max_zero_fuel             = 75000.
-    vehicle.mass_properties.cargo                     = 10000.  * Units.kilogram   
-    vehicle.mass_properties.payload                   = 16300.  * Units.kilogram   
-    vehicle.mass_properties.max_payload               = 20500. 
-
+    vehicle.mass_properties.max_takeoff               = 79015.8   # kg
+    vehicle.mass_properties.takeoff                   = 79015.8   # kg
+    vehicle.mass_properties.operating_empty           = 62746.4   # kg
+    vehicle.mass_properties.takeoff                   = 79015.8   # kg
+    vehicle.mass_properties.max_zero_fuel             = 62732.0   # kg #0.9 * vehicle.mass_properties.max_takeoff
+    vehicle.mass_properties.cargo                     = 0.       #10000.  * Units.kilogram   
+    
+    vehicle.mass_properties.center_of_gravity         = [60 * Units.feet, 0, 0]  # Not correct
+    vehicle.mass_properties.moments_of_inertia.tensor = [[10 ** 5, 0, 0],[0, 10 ** 6, 0,],[0,0, 10 ** 7]] # Not Correct
```

Comment:
Now included cargo weight. Updated weights regression to include this. Also, now CG calculated in 737 file